### DISCUSSION TITLE
How to draw  OpenBCI_GUI Time Serials

### DISCUSSION THREAD

**Discussion by programClown** [Live-Charts/LiveCharts2#1598] [Q&A]
Hi,
I'm building a avalonia App that analysis EEG data, but I can't draw Time Serials like OpenBCI_GUI shows.

![pic](https://github.com/OpenBCI/OpenBCI_GUI/blob/master/images/GUI-V4-Screenshot.jpg)

Thanks.

**Comment by beto-rodriguez** [comment]
Hello, I will move this to the Q&A section, please let me see what you have tried where are you stuck at, this way is easier for me to help you.
